src/component_builder/__main__.py

```diff
@@ -36,9 +44,15 @@ def cli():
     elif arguments['test']:
         build.run('test', components)
     elif arguments['tag']:
-        github.create_tag(os.environ['RELEASE_TAG'])
+        for comp in components:
+            github.create_tag(comp.env['VERSION'])
```

Comment:
This is an interesting situation, for single-component repos like this one, where you'd prefer the tag not to include the component's name.